### DISCUSSION TITLE
design_situations_demo - high-level-functions-demo

### DISCUSSION THREAD

**Discussion by MScEngineeringBV** [dlubal-software/Dlubal_JavaScript_Library#62] [General]
Hello,

If i run many of the high level demos, it returns an error message, i,e.

The script finished with error:
Error: design_situations[4] heeft niet de eigenschap SetCombinationWizard.

Stack trace:
() at 13

**Comment by jarabroz** [comment]
Hello,
I'm unable to reproduce it.
